source/Octopus.Versioning.Tests/Octopus/OctopusVersionTests.cs

```diff
@@ -777,15 +804,6 @@ public void TryParseTest()
             "",
             "",
             "")]
-        [TestCase("v0.0.1",
-            0,
-            0,
-            1,
-            0,
-            "",
-            "",
-            "",
-            "")]
```

Comment:
How did these test cases work if we didn't have support for a leading `v` before?

Reply:
These were in the "Invalid semver" tests, I've moved them to the valid test.

Reply:
Ahh I see, the GitHub diff hid that away.